app/src/test/java/com/vero/tasky/agenda/data/local/dao/EventDaoFake.kt

```diff
@@ -0,0 +1,38 @@
+package com.vero.tasky.agenda.data.local.dao
+
+import com.vero.tasky.agenda.data.local.EventWithPhotosAndAttendees
+import com.vero.tasky.agenda.data.local.entities.AttendeeEntity
+import com.vero.tasky.agenda.data.local.entities.EventEntity
+import com.vero.tasky.agenda.data.local.entities.RemotePhotoEntity
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+
+class EventDaoFake : EventDao {
+
+    private val eventList = mutableSetOf<EventWithPhotosAndAttendees>()
```

Comment:
When you add Attendees, delete event, etc, I would modify this eventList

So then you can verify using this eventlist